workloads/router-perf-v2/ingress-performance.sh

```diff
@@ -69,6 +69,7 @@ if [[ ${ENABLE_SNAPPY_BACKUP} == "true" ]] ; then
  snappy_backup "csv json" "http-perf.yml" "router-perf-v2"
 fi
 
+env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" WORKLOAD="$WORKLOAD" ES_SERVER="$ES_SERVER" ../../utils/index.sh
```

Comment:
there isnt any JOB_START or JOB_END variable here, think JOB_STATUS value can be replaced with ${result}

Reply:
so do i just eliminate those vars and do the replacement and things should work?

Reply:
You'll have to add the creation of those vars in the script to pass to index.sh

Reply:
just like this: 
https://github.com/cloud-bulldozer/e2e-benchmarking/blob/ae79591659203877f620a4fa69bc3013a0433f76/workloads/kube-burner-ocp-wrapper/run.sh#L129 
https://github.com/cloud-bulldozer/e2e-benchmarking/blob/ae79591659203877f620a4fa69bc3013a0433f76/workloads/kube-burner-ocp-wrapper/run.sh#L132